### PR TITLE
40 see consequences

### PR DESCRIPTION
**Pull request by Sarvesh-Mk**

### CODE REVIEW

File: devtools/debug.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def debug(*args, sep=' ', end='\n'):
             text += sep
     text += end
 
-    with open('logs/debug_logs.txt', 'a') as file:
+    with open('debug_logs.txt', 'a') as file:
```
Comment (yashkir) [inline devtools/debug.py]: remove this change

File: logic/StateManager.py [hunk 2]
```diff
@@ -52,7 +52,7 @@ def apply_stats(self, response):
         # TODO still more pork to cut here
         responses = self.game.situation_manager.current_situation.get_option_response()
         for r in range(len(responses)):
-            if response == responses[r]:
+            if int(response[0])-1 == r:
```
Comment (yashkir) [inline logic/StateManager.py:55]: whats the point of this change?

File: logic/situation.py [hunk 3]
```diff
@@ -26,4 +27,10 @@ def get_option_stats(self, index) -> dict:
...
     def get_option_stats_str(self, index) -> dict:
         """Returns the option stats as a string"""
-        return str(self.options[index]['stats'])
+        l_str = ''
+        l_str += ' ('
+        for x in self.options[index]['stats']:
+            l_str += (' ' + str(x) + ': ')
+            l_str += str(self.options[index]['stats'][x]) + ','
+        l_str += (')')
+        return l_str
```
Comment (yashkir) [inline logic/situation.py:34]: Use a format string here

File: screens/game_screen.py [hunk 4]
```diff
@@ -37,19 +39,21 @@ def update_text(self):
...
+                    str(r+1) + ' ' + str(response_list[r])
+                    + self.situation_manager.current_situation.get_option_stats_str(r),
+                    self.__choice), ('given', self.button_width)))
 
-        final_buttons = urwid.MonitoredFocusList(list_buttons, focus=0)
+        list_buttons.append((self.button_one, ('given', self.button_width)))
+        list_buttons.append((self.button_two, ('given', self.button_width)))
 
-        self.button_columns._set_contents(final_buttons)
+        # final_buttons = urwid.MonitoredFocusList(list_buttons, focus=0)
```
Comment (yashkir) [inline screens/game_screen.py]: delete this commented line

**Review by yashkir — CHANGES_REQUESTED** [review]

File: logic/StateManager.py (hunk 2 above)
Comment (Sarvesh-Mk) [reply logic/StateManager.py:55]: it makes list indexing easier

**Review by yashkir — APPROVED** [review]

**Pull request merged**